q2_quality_control/tests/test_decontam.py

```diff
@@ -115,6 +115,63 @@ def test_combined(self):
                 expecter_table = biom.Table.from_tsv(th, None, None, None)
             self.assertEqual(test_table, expecter_table)
 
+    def test_freq_prev_control_col_error_raised(self):
+        with self.assertRaises(ValueError) as context:
+            decontam_identify(
+                table=self.asv_table,
+                metadata=self.metadata_input,
+                method='frequency',
+                freq_concentration_column='quant_reading',
+                prev_control_column='Sample_or_Control')
+        self.assertIn("--prev-control-column", str(context.exception))
+
+    def test_freq_prev_control_col_indic_error_raised(self):
+        with self.assertRaises(ValueError) as context:
+            decontam_identify(
+                table=self.asv_table,
+                metadata=self.metadata_input,
+                method='frequency',
+                freq_concentration_column='quant_reading',
+                prev_control_indicator='Control Sample')
+        self.assertIn("--p-prev-control-indicator", str(context.exception))
+
+    def test_freq_params_error_raised(self):
+        with self.assertRaises(ValueError) as context:
+            decontam_identify(
+                table=self.asv_table,
+                metadata=self.metadata_input,
+                method='frequency')
+        self.assertIn("--p-freq-concentration-column", str(context.exception))
+
+    def test_prev_params_no_indic_error_raised(self):
```

Comment:
This method name says that we are testing that a certain type of error does not get raised but the method itself asserts that an error is raised.

Reply:
ah okay I changed the name of the test to be more clear, addressed in commit 82acad4b955d831983ba4ee95b4450d0f1109e4b